config/database.yml

```diff
@@ -12,7 +12,7 @@ defaults: &defaults
   timeout: 5000
   reconnect: true
   connect_timeout: 2
-  keepalives_idle: 30
+  keepalives_idle: 10
   keepalives_interval: 10
   keepalives_count: 2
   checkout_timeout: 5
```

Comment:
While we're here, how about adding a `statement_timeout` as well? Not sure what an appropriate setting would be. This guide suggests 250 (ms): https://github.com/ankane/the-ultimate-guide-to-ruby-timeouts#postgresql

Reply:
I thought about that, and I'm not against it. I was focused on timeouts that come about due to network-related issues. I can check the NewRelic stats to see what a reasonable time might be for identity-idp.

Reply:
Added a `statement_timeout` of 2.5 seconds. Looking at data in NewRelic, the longest statement we found ran for 1.5 seconds, so I padded it a bit.